contracts/base/Dispatcher.sol

```diff
@@ -177,12 +177,12 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, V4SwapRout
                     } else if (command == Commands.WRAP_ETH) {
                         // equivalent: abi.decode(inputs, (address, uint256))
                         address recipient;
-                        uint256 amountMin;
+                        uint256 amount;
```

Comment:
whats the name change motivation?

Reply:
the parameter is not a slippage check "minimum amount to wrap"... its just an amount to wrap. For unwrap and sweep it _is_ a slippage check so i think the naming must have been copied across. For wrapping its just an amount.